wordpress-mobile/WordPress-iOS
# Jetpack Focus: Update raw screenshots

Part of #18960

## Description
- Added new mocks to API-Mocks
- Updated raw screenshots generated in the `JetpackScreenshotTarget` to match new design specs (ref: pcdRpT-z4-p2#comment-731)
- Refactored the `removeApp()` method to accept an app name as a parameter

## Note
- ⚠️ This PR doesn't handle iPads yet. The notification screenshot flow fails on iPads -- I'll address this in a separate PR

## How to test

1. Run `rake mocks`
2. Switch to the `JetpackScreenshotGeneration` target
3. Run `testGenerateScreenshots()` on an iPhone simulator
4. ✅ Verify: tests succeed and the screenshots match design specs
5. Run `testGenerateScreenshots()` again
6. ✅ Verify: tests succeed and the screenshots match design specs

1 | 2 | 3 | 4 | 5 
-- | -- | -- | -- | --
![Simulator Screen Shot - iPhone 13 - 2022-07-21 at 15 37 24](https://user-images.githubusercontent.com/6711616/180252280-d291f75a-5ad5-4ea4-82b4-40f8905f9d47.png) | ![Simulator Screen Shot - iPhone 13 - 2022-07-21 at 15 37 32](https://user-images.githubusercontent.com/6711616/180252285-e6f3710a-ae97-4d49-a5f5-e7b0926c923c.png) | ![Simulator Screen Shot - iPhone 13 - 2022-07-21 at 15 37 38](https://user-images.githubusercontent.com/6711616/180252287-e2f2e60f-2a46-484b-abcd-fccc1dfe2344.png) | ![Simulator Screen Shot - iPhone 13 - 2022-07-21 at 15 37 44](https://user-images.githubusercontent.com/6711616/180252299-eb5a1a74-aec4-41d6-a123-b0bf1d28872b.png) | ![Simulator Screen Shot - iPhone 13 - 2022-07-21 at 15 40 07](https://user-images.githubusercontent.com/6711616/180252303-8767c389-1f54-4a55-9e6b-9bfea82b3db1.png)


## Regression Notes
1. Potential unintended areas of impact
- WordPressScreenshotGeneration

2. What I did to test those areas of impact (or what existing automated tests I relied on)
- Ran the WordPressScreenshotGeneration target manually

3. What automated tests I added (or what prevented me from doing so)
- n/a

PR submission checklist:
